src/source/Ice/IceAgent.c

```diff
@@ -490,6 +490,7 @@ STATUS iceAgentSendPacket(PIceAgent pIceAgent, PBYTE pBuffer, UINT32 bufferLen)
     // can be invoking incomingDataHandler and cause deadlock.
     // pIceAgent->lock has to be non-reentrant !!
     MUTEX_UNLOCK(pIceAgent->lock);
+    locked = FALSE;
```

Comment:
did someone just forget to set `locked = FALSE` my thread sanitizer complains about unlocking unlocked lock in `Cleanup`